### ISSUE TITLE
scroll bar covers the ... menu

### ISSUE THREAD

**Issue by rexxmagnus** [nextcloud/android#13139]
### ⚠️ Before posting ⚠️

- [X] This is a **bug**, not a question or an enhancement.
- [X] I've [searched for similar issues](https://github.com/nextcloud/android/issues) and didn't find a duplicate.
- [X] I've written a clear and descriptive title for this issue, not just "Bug" or "Crash".
- [X] I agree to follow Nextcloud's [Code of Conduct](https://nextcloud.com/contribute/code-of-conduct/).

### Steps to reproduce

Scroll in file selector so the index letter popout appears.
Try to click "..." Button next to a file

### Expected behaviour

Should be able to open menu

### Actual behaviour

The button is not touchable 

### Android version

10

### Device brand and model

Blackview bv900

### Stock or custom OS?

Stock

### Nextcloud android app version

3.29.0

### Nextcloud server version

29.0.0

### Using a reverse proxy?

No

**Comment by joshtrichards** [comment]
Pretty subtle, but true. Not sure this is a bug though. The nearest file entry's file menu is covered by the extra index letter scroll bar for ~1 second.  Then everything goes back to normal. : 🤔

**Comment by rexxmagnus** [comment]
Yes, it does go away, but this only works for files that are not the first in the list. If it's the first file in the list you want to click, touching the "..." zone actually brings up the letter and scrolling functionality. I suspect this would also happen with the last item, since the scrollbar is always going to be next to the last or first in the list.

**Comment by alperozturk96** [comment]
https://github.com/nextcloud/android/assets/67455295/9ed4ad44-58ea-45f5-8cf6-37d1a9f3c2a8

**Comment by alperozturk96** [comment]
| Old | New |
| ------------- | ------------- |
| ![old](https://github.com/nextcloud/android/assets/67455295/a14a3ccf-069b-4e78-a9b9-579ab6fa3f83) | ![new](https://github.com/nextcloud/android/assets/67455295/d7cafb0b-1b8e-431d-88b6-4ef8c5b8b14f)  |

While trying to fix this issue, I adjusted the margins and zIndex. The result was a new look, which I found more appealing.

 @nextcloud/designers What do you think? @jancborchardt

**Comment by jancborchardt** [comment]
@alperozturk96 nice on trying it out, but it diesn't look quite native, and a bit overly complex. We should rather go with native Android lists so people feel at home. :)

**Comment by alperozturk96** [comment]
[This](https://github.com/zhanghai/AndroidFastScroll?tab=readme-ov-file) 3rd party library used for this feature. When the side scroll bar appears, the onClick event is not triggered immediately. Users must wait for the auto-hide animation (1.5 second) to complete. Unfortunately, due to library limitations, we cannot fix this without disrupting the UI. Adding extra padding for the trailing icons is an option, but it would make the file list look odd.

Changing zIndex didn't work
Using bringFront() function didn't work
Using FixOnItemTouchListenerRecyclerView didn't work as well

**Comment by jancborchardt** [comment]
@alperozturk96 @tobiasKaminsky is there no native scroll bar library for this? Or would it be so bad if we just use the native scroll bar here without that extra functionality, if it interferes with more basic things like the actions?
